react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,22 @@
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.input {
+  display: flex;
+}
+
+i {
+  position: relative;
+  top: 20px;
+}
+
+p,
+.fa-times {
+  color: red;
+}
+
+.fa-check {
+  color: green;
+}
```

Comment:
It is bad practice to directly change the behavior of a class from a CSS library. This can cause confusion with other people working on the project or even yourself later on, if the class doesn't behave as expected. A better approach is to use classes you define to make the desired changes in conjunction with the classes from the CSS library. For example her you could do:

```CSS
.text-red {
  color: red;
}

.text-green {
  color: green;
}
```